src/openambit-cli/.gitignore

```diff
@@ -0,0 +1 @@
+personal_config.h
```

Comment:
is this gitignore useful in general or just a leftover?

Reply:
Just a leftover. I used to store a different APPKEY there, because on Task.cpp I had:

```
+#include "personal_config.h"
+++ b/src/openambit-cli/Task.cpp
@@ -7,8 +7,11 @@
 #include <movescount/logstore.h>
 #include <movescount/movescountxml.h>
 #include <libambit_int.h>
+#include "personal_config.h"
 
-#define APPKEY                 "HpF9f1qV5qrDJ1hY1QK1diThyPsX10Mh4JvCw9xVQSglJNLdcwr3540zFyLzIC3e"
+#ifndef APPKEY
+  #define APPKEY                 "HpF9f1qV5qrDJ1hY1QK1diThyPsX10Mh4JvCw9xVQSglJNLdcwr3540zFyLzIC3e"
+#endif
 #define MOVESCOUNT_DEFAULT_URL "https://uiservices.movescount.com/"

```